spec/features/recipes/show_spec.rb

```diff
@@ -0,0 +1,129 @@
+require "rails_helper"
+
+RSpec.describe "/recipes/:id", type: :feature do
+  before(:each) do
+    @ingredient_1 = Ingredient.create!(name: "Lemon", cost: 2)
+    @ingredient_2 = Ingredient.create!(name: "Asparagus", cost: 1)
+    @ingredient_3 = Ingredient.create!(name: "Salmon", cost: 10)
+
+    @ingredient_4 = Ingredient.create!(name: "Sauce", cost: 4)
+    @ingredient_5 = Ingredient.create!(name: "Spaghetti", cost: 2)
+    @ingredient_6 = Ingredient.create!(name: "Ground Beef", cost: 7)
+
+    @ingredient_7 = Ingredient.create!(name: "Salt", cost: 4)
+    @ingredient_8 = Ingredient.create!(name: "Pepper", cost: 4)
+    @ingredient_9 = Ingredient.create!(name: "Avocado Oil", cost: 4)
+
+    @recipe_1 = Recipe.create!(name: "Tangy Salmon", complexity: 5, genre: "Mediterranean")
+    @recipe_2 = Recipe.create!(name: "Tried and True", complexity: 2, genre: "Italian")
+    @recipe_3 = Recipe.create!(name: "Avo Asparagus", complexity: 1, genre: "Sides")
+
+    # Tangy Salmon
+    RecipeIngredient.create!(recipe_id: @recipe_1.id, ingredient_id: @ingredient_1.id)
+    RecipeIngredient.create!(recipe_id: @recipe_1.id, ingredient_id: @ingredient_2.id)
+    RecipeIngredient.create!(recipe_id: @recipe_1.id, ingredient_id: @ingredient_3.id)
+    RecipeIngredient.create!(recipe_id: @recipe_1.id, ingredient_id: @ingredient_7.id)
+    RecipeIngredient.create!(recipe_id: @recipe_1.id, ingredient_id: @ingredient_8.id)
+    RecipeIngredient.create!(recipe_id: @recipe_1.id, ingredient_id: @ingredient_9.id)
+
+    # Tried and True
+    RecipeIngredient.create!(recipe_id: @recipe_2.id, ingredient_id: @ingredient_4.id)
+    RecipeIngredient.create!(recipe_id: @recipe_2.id, ingredient_id: @ingredient_5.id)
+    RecipeIngredient.create!(recipe_id: @recipe_2.id, ingredient_id: @ingredient_6.id)
+    RecipeIngredient.create!(recipe_id: @recipe_2.id, ingredient_id: @ingredient_7.id)
+    RecipeIngredient.create!(recipe_id: @recipe_2.id, ingredient_id: @ingredient_8.id)
+
+    # Avo Asparagus
+    RecipeIngredient.create!(recipe_id: @recipe_3.id, ingredient_id: @ingredient_1.id)
+    RecipeIngredient.create!(recipe_id: @recipe_3.id, ingredient_id: @ingredient_2.id)
+    RecipeIngredient.create!(recipe_id: @recipe_3.id, ingredient_id: @ingredient_7.id)
+    RecipeIngredient.create!(recipe_id: @recipe_3.id, ingredient_id: @ingredient_8.id)
+    RecipeIngredient.create!(recipe_id: @recipe_3.id, ingredient_id: @ingredient_9.id)
```

Comment:
Woweeeee you were right, that's lots of objects! 

I suggest limiting yourself to 2-3 of each objects maximum for ICs like this. That is usually plenty. However, this is great to see you have the ability to set up this ..magnitude of objects!